docs/dictionaries.md

```diff
@@ -101,28 +108,33 @@ Terms from the National Cancer Institute Thesaurus for healthy control status.
 }
 ```
 
-The `IsAbout` relation uses a term from the neurobagel (`bg`) namespace because
+The `IsAbout` relation uses a term from the Neurobagel namespace because
 `"Diagnosis"` is a standardized term.
 
 ## Sex
 
-Terms are from the BIDS recommended namespace.
+Terms are from the SNOMED-CT ontology.
```

Comment:
maybe mention that SNOMED-CT has controlled terms, and they align with BIDS? (definitely :cherries:)